Kombit.Samples.CH.WebsiteDemo/X509CertificateSigningSecurityTokenResolverFactory.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Safewhere.ServiceProvider;
+using Safewhere.ServiceProvider.Config;
+using Safewhere.ServiceProvider.Util;
+using System;
+using System.IdentityModel.Selectors;
+
+namespace WebsiteDemo
+{
+    /// <summary>
+    /// A factory which returns X509CertificateSigningSecurityTokenResolverFactory for signing key resolver.
+    /// </summary>
+    public class X509CertificateSigningSecurityTokenResolverFactory : DefaultSecurityTokenResolverFactory
```

Comment:
The websitedemo project uses nita.saml2.0 to read SAML assertions. Does it need these resolvers? I thought it doesn't.